packages/v4/patternfly-docs/content/accessibility/card/card.md

```diff
@@ -3,21 +3,102 @@ id: Card
 section: components
 ---
 
-A **card** is a square or rectangular container that can contain any kind of content. Cards symbolize units of information, and each one acts as an entry point for users to access more details. For example, in dashboards and catalog views, cards function as a preview of a detailed page. Cards may also be used in data displays like card views, or for positioning content on a page.
-
-Basic cards will have native keyboard interaction with **Tab** navigating between interactive elements and **Shift + Tab** navigating backwards. Any elements inside a card will follow their usual accessibility expectations. 
-
-<br/>
-In general, the card component already has accessibility built in. However, variants that will need additional considerations are:
-- **Expandable variant:** for expandable variants it’s important to add additional details for screen reader users to understand this interaction. 
-Add:
-
-  ```
-    toggleButtonProps=({
-      id: ‘toggleId’, 
-      ‘aria-label’: ‘toggleLabel’, 
-      ‘aria-labelledby’: ‘titleId toggleId’, 
-      ‘aria-expanded’: this.state.isExpanded
-    )}
-  ```
-  Note in this example that the `aria-labelledby` is a combination of the id on the `CardTitle` and the id on the toggle button. This will then combine the card title and the toggle button's label (this is the `aria-label` with text defining the toggle button, like "Details") and create something like "Header Details, collapsed, button" for the screen reader user. These are highly recommended for accessibility.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+## Accessibility
+
+To implement an accessible PatternFly **card**:
+
+- Provide an accessible name for the card via the `aria-label` attribute, or by linking to a card title via the `aria-labelledby` attribute.
+- Ensure any other PatternFly components within a card follow their own accessibility documentation.
+- Place cards inside of a list element whenever displaying a set of multiple cards, with a list item wrapped around each card. This list element must also have its own accessible name via the `aria-label` attribute if there is no other surrounding context.
+
+## Testing
```

Comment:
We should probably include testing for a selectable card. I imagine that we could pull this [selectable card a11y content from the examples page](https://www.patternfly.org/v4/components/card#selectable-card-accessibility-features) and place it on this a11y page? Or should we keep it on the examples page?

Reply:
Honestly I think it can work on either page (or both). We could trim down the example verbiage a little and move some of it over to the a11y page, so that the example doesn't get *as* into the a11y reasoning behind using those props.

I'm not opposed to that, but would a selectable card be something that would be considered the "minimum" for cards, or something that isn't exactly a "standard" Card and more something consumers would continue reading in the React Customization section for their particular use-case?

Reply:
I think I was imagining adding one check for testing that kind of variant, but I could see us also just including it further down like we do with [Toast Alerts](https://www.patternfly.org/v4/components/alert/accessibility#toast-alerts) since it's a specific variant. I guess that's a good question - should we include testing for specific variants or just include a separate section? Have we already talked about that before?

Reply:
Yeah, I think it makes sense on thinking about it to add one more testing step that starts with clear context (like the ones that start with 'If there's no surrounding context...'
The selectable testing step could start with 'If building a selectable card....' or something like that

Reply:
Sounds good! I'm not sure if it was discussed previously, but moving forward it can make sense to add a checkbox test for these kinds of variants. Each section would still kinda build upon the previous in a "No information about specific variant > A short mention to test for a specific variant > More details behind the prop/attribute for the variant > even more info if needed about the variant" sort of way.